src/swap/components/SwapAmountInput.tsx

```diff
@@ -37,6 +40,17 @@ export function SwapAmountInput({ label, token, type }: SwapAmountInputReact) {
     return setFromToken;
   }, [type, setFromToken, setToToken]);
 
+  const balanceResponse: UseBalanceReturnType = useBalance({
```

Comment:
<img width="658" alt="Screenshot 2024-06-13 at 2 24 23 PM" src="https://github.com/coinbase/onchainkit/assets/132851666/30925dbc-38ed-4217-9d28-ce02274d7582">

think this might only work with eth

Reply:
@abcrane123 let's refactor this later today. Thank you @kyhyco